src/main/java/io/sphere/sdk/meta/ProductAttributeDocumentation.java

```diff
@@ -0,0 +1,178 @@
+package io.sphere.sdk.meta;
+
+import io.sphere.sdk.attributes.AttributeAccess;
+import io.sphere.sdk.products.ProductVariantDraftBuilder;
+
+/**
+ <h3 id="intro">Introduction</h3>
+ <h3 id="product-type-creation">ProductType Creation</h3>
+ <p>A {@link io.sphere.sdk.producttypes.ProductType} is like a schema how the product attributes are structured.</p>
+ <p>{@link io.sphere.sdk.producttypes.ProductType}s contain a list of {@link io.sphere.sdk.attributes.AttributeDefinition}s, which can be seen as the attribute name and the corresponding type of the attribute.
```

Comment:
I would say:
"... which corresponds to the name and type of each attribute, along with some additional information".